nemo/collections/nlp/data/datasets/sgd_dataset/prediction_utils.py

```diff
@@ -24,10 +24,161 @@
 
 REQ_SLOT_THRESHOLD = 0.5
 
-__all__ = ['get_predicted_dialog', 'write_predictions_to_file']
+__all__ = ['get_predicted_dialog_baseline', 'write_predictions_to_file']
 
 
-def get_predicted_dialog(dialog, all_predictions, schemas):
+def get_predicted_dialog_ret_sys_act(dialog, all_predictions, schemas, eval_debug):
+    """Update labels in a dialogue based on model predictions.
+  Args:
+    dialog: A json object containing dialogue whose labels are to be updated.
```

Comment:
could you please add comments about the retrieving slots from the system utterance

Reply:
Done